chapter2/AppCreationScripts/sample.json

```diff
@@ -10,7 +10,7 @@
     "AADApps": [
       {
         "Id": "service",
-        "Name": "TodoListAPI",
+        "Name": "TodoListAPI-chapter2",
```

Comment:
>-chapter2 [](start = 28, length = 9)

if the same app reg is to be used in chapters 1 and 2 , then this distinction is required, otherwise revert